src/App.js

```diff
@@ -12,6 +12,7 @@ class App extends React.Component {
     this.state = {
       recordInfo: getRecordInfo(getData().data, getData().planInfo),
     };
+    console.log(this.state.recordInfo.data);
```

Comment:
Probably the line is only for the reviewer's benifits. I would recommend using setting breaking points in the debugging tool 😄

Reply:
Oh gosh, you're right. I had that in there when I was checking something and completely missed removing it again 🙈